backend/api/management/commands/seed.py

```diff
@@ -175,7 +201,9 @@ def _create_annotation_campaigns(self):
         self.campaigns = []
         for campaign_data in campaigns:
             campaign = AnnotationCampaign.objects.create(
```

Comment:
Could we have a campaign without IndicatorSets (whichever one you want) ? Ideally the seeding should create data corresponding to every situation possible so that we can easily test things during dev :wink: